.travis.yml

```diff
@@ -166,6 +163,12 @@ script:
           gdb -return-child-result -batch -ex r -ex bt --args python $PYTHON_ARGS tests.py $NOSE_ARGS $TEST_ARGS
         fi
       else
+        # Workaround for pytest-xdist flaky colletion order
+        # https://github.com/pytest-dev/pytest/issues/920
+        # https://github.com/pytest-dev/pytest/issues/1075
+        export PYTHONHASHSEED=$(shuf -i 1-4294967295 -n 1)
+        echo PYTHONHASHSEED=$PYTHONHASHSEED
```

Comment:
Well, I have find out that xdist collection problem is triggered by `test_mathtext.py`, but I do not know what exactly, maybe `globals()` injection magic or something else. I think the best option for us is to rewrite `test_mathtext` to pytest fixtures and forget about this xdist issue.